src/thesis.tex

```diff
@@ -72,14 +73,16 @@
 % Doctrine of classification as it states the Socrates-Erasmus:
 \dziedzina{  
 11.3 Informatyka\\
+% TODO the above in English
 }
 
 % TODO
 % Subject classification due to ACM
 \klasyfikacja{D. Software}
 
 % Keywords list
-\keywords{system, aws, cost, optimization, heuristic, cost management, koszt, aws, optymalizacja, heurystyka kosztowa}
+\keywords{cloud, system, AWS, cost, optimization, heuristic, cost management}
```

Comment:
cloud -> could computing
cost,optimization-> cost optimization 
AWS-> Amazon Web Services (AWS) 
delete:  system, cost, (too broad)

Reply:
these have to comprehensively explain the thesis' aim afaik

Reply:
so probably we'll need many more of these there - we'll probably discuss that tomorrow